### PR TITLE
Request to join the meeting

### PR DESCRIPTION
**Pull request by Hanks10100**
I am a developer of Weex and have a great interest in W3C standards.

### CODE REVIEW

File: meetings/2018-11-17.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 * 周鹏飞（美团点评）
 * 尹立（百度）
 * 刘思杨（百度）
+* 张翰（阿里巴巴）
```
[the remark this answers is not in the document]
Comment (Honry) [reply meetings/2018-11-17.md]: Do you want to add these two participants as well?

[the remark this answers is not in the document]
Comment (Honry) [reply meetings/2018-11-17.md]: @gsimcard, please confirm then I could merge it.

**Comment by Honry** [comment]
Confirmed by @tayqassqan. Thanks!

File: meetings/2018-11-17.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gsimcard) [reply meetings/2018-11-17.md]: Yes, please. Thanks. @Honry

**Pull request merged**